applications/ultrasound_postprocessing/ultra_post/filters/registry.py

```
@@ -0,0 +1,85 @@
+    "median_filter": {"size": 3},
+    "gaussian_filter": {"sigma": 1.0},
+    "unsharp_mask": {"sigma": 1.0, "amount": 1.5, "threshold": 0.0},
+    "gamma_compression": {"gamma": 1.0},
+    "guided_filter": {"radius": 5, "eps": 0.01, "use_luminance": True},
+    "non_local_means": {"h": 0.1, "patch_size": 7, "patch_distance": 11, "fast_mode": True},
+    "svd_denoise": {"rank": 32, "suppress": "low", "shrink": 0.0},
+    "persistence": {"alpha": 0.5},
+    "temporal_svd": {"history": 5, "rank": 3},
+}
+
+__all__ = ["FILTERS", "DEFAULT_PARAMS"]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)